### DISCUSSION TITLE
3Dev 1.1.0

### DISCUSSION THREAD

**Discussion by aw4yout** [1Kuso4ek1/3Dev#7] [General]
### Major changes
- Sound manager class
- Renderer class for easy PBR initialization
- Simplified Model, Shape, Material, SceneManager and ShadowManager constructors and usage
### Minor changes
- Fixed a bug with shadows on Windows

<hr /><em>This discussion was created from the release <a href='https://github.com/1Kuso4ek1/3Dev/releases/tag/v1.1.0'>3Dev 1.1.0</a>.</em>

**Comment by SERJCOM** [comment]
хуйня

**Reply by aw4yout** [reply]
согл
